WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -191,13 +190,13 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         mDay = c.get(Calendar.DAY_OF_MONTH);
         mHour = c.get(Calendar.HOUR_OF_DAY);
         mMinute = c.get(Calendar.MINUTE);
-        mCategories = new ArrayList<String>();
+        //mCategories = new ArrayList<String>();
```

Comment:
Do you mind adding a TODO here? I don't think we'll forget it, but just in case.